clang/lib/Driver/Driver.cpp

```diff
@@ -4838,12 +4844,11 @@ class OffloadingActionBuilder final {
         for (auto TargetActionInfo :
              llvm::zip(SYCLDeviceActions, SYCLTargetInfoList)) {
           Action *&A = std::get<0>(TargetActionInfo);
-          auto &TargetInfo = std::get<1>(TargetActionInfo);
```

Comment:
Can you put this line back to reduce the possibility of merge conflicts?

Reply:
This was added [here](https://github.com/intel/llvm/commit/9152ef89097daf1b58b1c007e73128b0c8c101c1#diff-f8c64e36dfe3828a6f816c4550e78bb0305769ace1be53207e86ac9a3280ac9e), not in upstream DPC++. It is not needed, so I dropped that.